lib/tictactoe/board.rb

```diff
@@ -7,14 +7,15 @@ def initialize()
     end
 
     def draw
-      first_row = " #{@positions[0]} | #{@positions[1]} | #{@positions[2]}\n"
-      separator = "---|---|---" + "\n"
-      second_row = " #{@positions[3]} | #{@positions[4]} | #{@positions[5]}\n"
-      third_row = " #{@positions[6]} | #{@positions[7]} | #{@positions[8]}\n"
-      space = "" + "\n"
-      board_separator = "---------------" + "\n"
-      body = first_row + separator + second_row + separator + third_row + space + board_separator
-      puts body
+      print " #{positions[0]} | #{positions[1]} | #{positions[2]} \n"
+      print "-----------\n"
+      print " #{positions[3]} | #{positions[4]} | #{positions[5]} \n"
+      print "-----------\n"
+      puts " #{positions[6]} | #{positions[7]} | #{positions[8]} "
+    end
+
+    def is_not_board_full
```

Comment:
Considering that this method is called on Board, you don't need to restate board again in the method name. `board.is_not_full` should suffice.